### PR TITLE
Exp use case

### PR DESCRIPTION
**Pull request by manuelPartida**
code to make the experimentation use case work!

### CODE REVIEW

File: app/routes.py [hunk 1]
```diff
@@ -28,15 +29,16 @@ def index():
...
-    all_flags = json.dumps(ldclient.get().all_flags(current_user.get_ld_user()))
-
-    beta_features = ldclient.get().variation('dark-theme', current_user.get_ld_user(), False)
+    beta_features = ldclient.get().variation('dark-theme', user, False)
     
     set_theme = '{0}/index.html'.format(current_user.set_path)
 
     return render_template(set_theme, title='Home',
-    display_widgets=display_widgets, all_flags=all_flags, show_beta=beta_features)
+    display_widgets=display_widgets, all_flags=all_flags, show_beta=beta_features, user=user)
```
Comment (levlaz) [inline app/routes.py]: You can get `current_user` directly from jinja in the template. Is there a reason why you are adding this here instead?

File: app/templates/partials/footer.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
     <script src="{{ url_for('static', filename='light/dist/js/sb-admin-2.js') }}"></script>
    <script>
 
-       document.getElementById("clickGoal").onclick = ldclient.track("chart-click");
+       //document.getElementById("clickGoal").onclick = ldclient.track("chart-click");
```
Comment (levlaz) [inline app/templates/partials/footer.html]: Is this commented out?

File: app/routes.py [hunk 3]
```diff
@@ -54,13 +56,21 @@ def darkTheme():
 
 @core.route('/experiments')
 def experiments():
+    
+    user_key = str(uuid.uuid1())
+    user = {'key': user_key}
```
Comment (levlaz) [inline app/routes.py]: why aren't you using `get_ld_user(random=True)` instead of this?

File: app/routes.py (hunk 1 above)
Comment (manuelPartida) [reply app/routes.py]: Yes, i had to do it this way in order to get the experimentation use case to work. From what i could tell current_user would always evaluate to who ever was logged into support service and so when we would call get_ld_user(True) in the experiments route, it wouldnt have the intended effect and i wasnt able to trigger the goal event. 

Once i explicitly called the user in the route and passed it to the JS initialization, it then worked as expected.

File: app/templates/partials/footer.html (hunk 2 above)
Comment (manuelPartida) [reply app/templates/partials/footer.html]: I may have disabled it, but dont know if it was working in other areas of the app. I wasnt using that code

File: app/routes.py (hunk 3 above)
Comment (manuelPartida) [reply app/routes.py]: Not in particular, i can update this.

**Pull request merged**